packages/server/graphql/types/EstimateStage.ts

```diff
@@ -109,6 +116,36 @@ const EstimateStage = new GraphQLObjectType<Source, GQLContext>({
             type: 'string'
           }
         }
+        if (service === 'gitlab') {
+          const {gid} = integration
+          const gitlabAuth = await dataLoader
+            .get('teamMemberIntegrationAuths')
+            .load({service: 'gitlab', teamId, userId: viewerId})
+          if (!gitlabAuth?.accessToken) return null
```

Comment:
-2 This is failing for meeting participants without gitlab integration. They will not be able to join a Poker Meeting with GitLab issues at all.
```
GraphQLError: Cannot return null for non-nullable field EstimateStage.(serviceField,serviceField,serviceField,serviceField,serviceField,serviceField,serviceField,serviceField).
```
Reproduce: Ingo has GitLab integration, creates a Poker Meeting, adds GitLab tasks and moves to estimate stage. Paul has no integration and tries to join the meeting.